.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+
+# This workflow can be manually triggered to create a GitHub release and upload release bins.
+#
+# Steps to use:
+# - Create a new Git tag and push
+# - Navigate to https://github.com/rmohr/bazeldnf/actions/workflows/release.yml
+# - Click on "Run workflow"
+# - Select the tag you want to release
+# - Click on green button
+#
+# Requires read-write workflow permissions which can be configured in Actions repo settings.
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bazelbuild/setup-bazelisk@v1
+      - run: bazel build -c opt //cmd
+      - run: |
+          mkdir -p dist
+          cp -L bazel-bin/cmd/cmd_/cmd dist/bazeldnf-linux-x86_64
+      - uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            ./dist/bazeldnf-linux-x86_64
```

Comment:
At least for kubevirt we will also need arm. Should be relatively easy to do since with rules_go we can just create a target like

```
go_binary(
    name = "bazeldnf-amd64",
    embed = [":go_default_library"],
    goarch = "amd64",
    goos = "linux",
    visibility = ["//visibility:public"],
)

go_binary(
    name = "bazeldnf-arm64",
    embed = [":go_default_library"],
    goarch = "arm64",
    goos = "linux",
    visibility = ["//visibility:public"],
)
```

Reply:
Since this patch uses an external linker (gcc frontend / bfd backend), I think this will require a cross-compiler with an arm64 crt0. At the very least, when trying these extldflags on macOS for a Linux build, I got an error.

Maybe it's fine to let the Go compiler use its native linker and dynamically import libc.so? For C/C++ apps this should be avoided but if Go only imports POSIX libc symbols it's probably fine to use default `go build` without netgo, -static, etc.

@leoluk Could you advise what's the best way forward here?